src/widgets/toggle/toggle.js

```diff
@@ -0,0 +1,206 @@
+import React, { render, unmountComponentAtNode } from 'preact-compat';
```

Comment:
This widget shouldn't be here, it belongs to the `develop` branch and has been renamed to `toggle-refinement` (the file should be deleted).